src/beautifiers/phpcbf.coffee

```diff
@@ -11,12 +11,10 @@ module.exports = class PHPCBF extends Beautifier
   isPreInstalled: false
 
   options: {
-    _:
-      standard: ["standard", (standard) ->
-        if (standard) then \
-          standard else "PEAR"
-      ]
-    PHP: true
+    PHP:
+      phpcbf_path: true
+      phpcbf_version: true
+      standard: true
```

Comment:
Sounds good. Let's set `PEAR` as default, so we are backwards compatible. Otherwise, this looks good to go!